### DISCUSSION TITLE
Decimal Precision for Tablet Area

### DISCUSSION THREAD

**Discussion by 19peytonsmith** [ppy/osu#15034] [Ideas] [answered]
Greetings, this might just be a minor issue that many might not be dealing with I think it would be beneficial to bring about the idea of introducing decimal precision for tablet area, specifically 2 decimal places, like other tablet drivers out there. I would like to copy my tablet area exactly as it is with my current tablet drivers but I can't, and it feels just slightly off. 

Thanks!

**Comment by peppy** [comment, marked as answer]
Just to confirm, you are looking for *sub milllimetre precision*? Can you please share what your area is to require this level of precision? It does sound like it could be placebo or something else causing your specified area to feel off.

**Reply by 19peytonsmith** [reply]
I have my area all the way in the top right corner. Width: 59.41mm, Height: 42.22mm, X:122.3mm, Y:21.11mm.

**Reply by 19peytonsmith** [reply]
Yeah it is not a big issue, but if others are in the same boat, it doesn't hurt to make the request.

**Reply by peppy** [reply]
Duplicate of https://github.com/ppy/osu/discussions/13188.
